### PR TITLE
feat: add support for egui version 0.28

### PR DESCRIPTION
**Pull request by PPakalns**

### CODE REVIEW

**Review by sgoudham — APPROVED** [review]
I'm happy with this if @Stonks3141 is.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "catppuccin-egui"
-version = "5.1.1"
+version = "5.1.2"
```
Comment (sgoudham) [inline Cargo.toml:3]: We use release-please to control the versioning of this repository, which is driven by the PR title (i.e. `feat` for minor, `fix` for patch)

Please revert this change.

```suggestion
version = "5.1.1"
```

**Review by sgoudham — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (PPakalns) [reply Cargo.toml]: Maybe comment could be added to Cargo.toml or contributing guide added to repository to avoid future problems!

Thanks for the fix, commited the change :)

**Review by sgoudham — APPROVED** [review]
Thanks! I think these changes should work so will merge.
